x-pack/plugins/apm/public/components/app/TransactionOverview/index.tsx

```diff
@@ -154,9 +159,45 @@ export function TransactionOverview() {
               <h3>Transactions</h3>
             </EuiTitle>
             <EuiSpacer size="s" />
+            {!transactionListData.isAggregationAccurate && (
+              <EuiCallOut
+                title={i18n.translate(
+                  'xpack.apm.transactionCardinalityWarning.title',
+                  {
+                    defaultMessage:
+                      'This view shows a subset of reported transactions.',
+                  }
+                )}
+                color="danger"
+                iconType="alert"
+              >
+                <p>
+                  <FormattedMessage
+                    id="xpack.apm.transactionCardinalityWarning.body"
+                    defaultMessage="The number of unique transaction names exceeds the configured value of {bucketSize}. Try reconfiguring your agents to group similar transactions or increase the value of {codeBlock}"
+                    values={{
+                      bucketSize: transactionListData.bucketSize,
+                      codeBlock: (
+                        <EuiCode>
+                          xpack.apm.ui.transactionGroupBucketSize
+                        </EuiCode>
+                      ),
+                    }}
+                  />
+
+                  <ElasticDocsLink section="/kibana" path="TODO">
```

Comment:
The docs link will be `/guide/troubleshooting.html#troubleshooting-too-many-transactions`. PR coming tomorrow (couldn't quite finish it today, sorry).
```suggestion
                  <ElasticDocsLink
                    section="/kibana"
                    path="/troubleshooting.html#troubleshooting-too-many-transactions"
                  >
```

Reply:
Perfect, thanks!